index.js

```diff
@@ -7,4 +7,5 @@ express()
   .set('views', path.join(__dirname, 'views'))
   .set('view engine', 'ejs')
   .get('/', (req, res) => res.render('pages/index'))
-  .listen(PORT, () => console.log(`Listening on ${ PORT }`))
+  .listen(process.env.PORT || 5000
```

Comment:
`PORT` is already dynamic (see line 3 in this file), so I don't think this is needed :-)